test/tests/providers/capi/capa-mimir/platform/atlas/alerting-rules/mimir.rules.test.yml

```diff
@@ -389,7 +389,7 @@ tests:
     input_series:
       # mimir-ingester real memory usage gradually decreases until it goes below 30% of the memory requests.
       - series: 'cortex_compactor_runs_failed_total{reason="error", installation="golem", cluster_id="golem", namespace="mimir", pipeline="testing", provider="capa"}'
```

Comment:
Not related to this PR but the comment looks completely unrelated to the metrics!

Reply:
Removed it